seq/alphabet_conv.go

```diff
@@ -521,118 +521,33 @@ func RNAToProtein(rna []byte, transl_table int) (protein []byte, err error) {
 		}
 	}
 	if !found {
-		return nil, errors.New("seq.RNAToProtein: translation table is not found")
+		return errors.New("seq.DNAToProtein: translation table is not found")
 	}
+	return nil
+}
 
-	var last3NTs bytes.Buffer
-	for _, nt := range rna {
+// DNAToProtein accepts a DNA Sequence, translation table from NIBH standard and returns a protein sequence
+func DNAToProtein(dna []byte, transl_table int) ([]byte, error) {
```

Comment:
Option `frame`?

For better handling reverse complement sequence, this function can be method of `type Seq`.